index.js

```diff
@@ -0,0 +1,16 @@
+let names=[]
+function myFunction()
+{
+ let x=document.getElementById("box");
+  names.push(document.getElementById("name").value);
+ x.innerHTML=names.join('<br/><br/>'); 
+}
+
+//for accessibility
+let input = document.getElementById("name");
+input.addEventListener("keypress", function(event) {
+  if (event.key === "Enter") {
+    event.preventDefault();
+    document.getElementById("add-btn").click();
```

Comment:
What does this last line of code do on line 14 do?

Reply:
It'll allow for the trigger of the HTML element with the ID of "add-btn" with the click event handler to act as if it was actually clicked ( say it's a click event simulator). This enables the user to add and display the data from the input by tapping the enter button